rush/rush/src/actions/RebuildAction.ts

```diff
@@ -66,11 +66,6 @@ export default class RebuildAction extends CommandLineAction {
   }
 
   protected onDefineParameters(): void {
-    this._quietParameter = this.defineFlagParameter({
-      parameterLongName: '--quiet',
-      parameterShortName: '-q',
```

Comment:
Maybe we should have this show a warning for the next few releases, to give people a little time to transition?